lib/containers/k8s.pm

```diff
@@ -21,22 +21,18 @@ use registration qw(add_suseconnect_product get_addon_fullname);
 our @EXPORT = qw(install_k3s uninstall_k3s install_kubectl install_helm);
 
 sub install_k3s {
-    my $k3s_dowload_url = get_required_var("K3S_DOWNLOAD_URL");
-    assert_script_run("curl -LO $k3s_dowload_url");
-    assert_script_run("install -c -m 744 ./k3s /usr/local/bin/k3s");
+    assert_script_run("curl -sfL https://get.k3s.io | sh -");
     enter_cmd("k3s server 2>&1 | tee k3s.log &");
-    script_retry("grep 'Wrote kubeconfig /etc/rancher/k3s/k3s.yaml' k3s.log", delay => 20, retry => 10);
+    script_retry("cat /etc/rancher/k3s/k3s.yaml", delay => 20, retry => 10);
     assert_script_run("k3s kubectl get node");
     script_run("mkdir \$HOME/.kube");
     script_run("rm \$HOME/.kube/config");
     assert_script_run("ln -s /etc/rancher/k3s/k3s.yaml \$HOME/.kube/config");
 }
 
 sub uninstall_k3s {
-    my $pid = script_output("ps | grep k3s-server | head -n 1 | cut -d ' ' -f1");
-    script_run("kill $pid");
     script_run("rm \$HOME/.kube/config");
-    script_run("rm /usr/local/bin/k3s");
+    script_run("/usr/local/bin/k3s-uninstall.sh");
```

Comment:
why not `assert_script_run` here?

Reply:
yes, agree, and just in case, with `timeout => xyz` provided `>90` in case it takes more than 90s to remove stuff.

Reply:
This is an interesting question, this is inside of a post_fail_hook and based on my  experience we don't assert_script_runs on post_fail_hook. I am going to wait other experienced mates about that

Reply:
I vote for `assert_script_run` as well.

Reply:
Thanks, now are `assert_script_run`